library/oraclelinux

```diff
@@ -4,10 +4,10 @@ GitCommit: 5c8a1c296acd6e90487cd261d16cf85fd6bcb73f
 GitFetch: refs/heads/master
 # https://github.com/oracle/container-images/tree/dist-amd64
 amd64-GitFetch: refs/heads/dist-amd64
-amd64-GitCommit: 2fec92641c29d7a4cc704014d081fa42816ec82f
+amd64-GitCommit: d2cddc02be12d3574ddf2553d94d62ef2d906db1
 # https://github.com/oracle/container-images/tree/dist-arm64v8
 arm64v8-GitFetch: refs/heads/dist-arm64v8
-arm64v8-GitCommit: 0b869f1e3a5cb823ace5ac25a0624e63d4d40f7a
+arm64v8-GitCommit: 0769017cd168c14629815f102a85fbb859a05793
```

Comment:
I think this is still somehow not quite right? https://github.com/oracle/container-images/commit/0769017cd168c14629815f102a85fbb859a05793 :grimacing:

Maybe this one was the intended commit?

```suggestion
arm64v8-GitCommit: a081765cbdaf3306c7f1ccc45dd525cea67a96b5
```

Reply:
Yes, cut-and-paste from wrong window again.  I hope to automate this process soon.  Thanks for your patience.